core/src/main/scala/org/http4s/ResponseCookie.scala

```diff
@@ -98,6 +98,7 @@ final case class ResponseCookie(
     maxAge: Option[Long] = None,
     domain: Option[String] = None,
     path: Option[String] = None,
+    sameSite: SameSite = SameSite.Lax,
```

Comment:
I wonder if this should be an `Option`, so we can round trip values, or continue to send cookies without this attribute.  Semantically, `Lax` is the default, but by keeping it optional, it would remain possible to send cookies without this attribute.

Reply:
By round trip do you mean that the cookie should be returned unchanged if nothing is done? In which case would I also need to remove this change: `if (secure || sameSite == SameSite.None) writer.append("; Secure")`.

Reply:
Ooh, yeah.  That's unfortunate, because as is, we're trying to do the right thing by default here.  And generally we make sure that we can go [model -> string -> model](https://github.com/http4s/http4s/blob/136ab44c439bec7c532518080fc619dfb7682f8a/laws/src/main/scala/org/http4s/laws/HttpCodecLaws.scala#L11), which is different from string -> model -> string.  This is probably better how you wrote it.

Reply:
If this behaviour is desired do you think it is worth writing an (explicit) test for it?